app/controller/InfoController.js

```diff
@@ -546,6 +546,46 @@ SDL.InfoController = Em.Object.create(
         );
     },
 
+    
+    startStreamingAdapter: function(url) {

```

Comment:
@LuxoftAKutsan please add a description for this

Reply:
bba3698